### DISCUSSION TITLE
Does Jellyfin support "season names" from TMDB?

### DISCUSSION THREAD

**Discussion by ghost** [jellyfin/jellyfin#10714] [Q&A] [answered]
e.g. in Plex, [JoJo's Bizarre Adventures (2012)](https://www.themoviedb.org/tv/45790/seasons) `Season 1` would display as `Phantom Blood & Battle Tendency`, `Season 2` would display as `Stardust Crusaders`, and so forth.

I see I can *manually* set season names in Jellyfin, but I don't see any way to automatically pick those up. I'm not going to manually do this for 100s of shows, so if it's not supported, then...oh well.

---

I did try searching GitHub issues/PRs/discussions for stuff about this, but I couldn't find anything super relevant. Maybe GitHub search just sucks.

**Comment by 1337joe** [comment, marked as answer]
Open the TMDB plugin settings page (Menu > Dashboard > Plugins > TMDb) and look for the setting labeled "Import season name from metadata fetched for series."

Make sure it's enabled (I think it defaults to off) and refresh metadata for a show that you expect it to work on. If it still doesn't work that would best be filed as a bug report.

**Reply by Shadowghost** [reply]
Also keep in mind that season names often exist in specific orders within tmdb and not on the default order. I am not sure we support this use case properly though.

**Reply by ghost** [reply]
Thank you @1337joe and noted @Shadowghost. 👍🏻

**Comment by tommargar** [comment]
With the TV Show 'Banshee' https://www.themoviedb.org/tv/41727-banshee/seasons and the NFO filled from Tiny Media Manager, the season titles do show up.

However, the season names are in alphabetical order, leadinng to season 2 being listed first int the season view. I've had to add 'S01 - ' ... manually to get it to show up usable in JF.
